extensions/csrf-reactive/runtime/src/main/java/io/quarkus/csrf/reactive/runtime/CsrfReactiveConfig.java

```diff
@@ -19,6 +19,12 @@ public class CsrfReactiveConfig {
     @ConfigItem(defaultValue = "csrf-token")
     public String formFieldName;
 
+    /**
+     * Token header which will provide a CSRF token.
+     */
+    @ConfigItem
+    public String tokenHeaderName;
```

Comment:
This should have a default value. Merely importing the CSRF extension now makes my build fail :(